ssl/base/src/main/java/org/wildfly/security/ssl/ElytronMessages.java

```diff
@@ -111,7 +110,7 @@ interface ElytronMessages extends BasicLogger {
     CertificateException emptyChainNotTrusted();
 
     @Message(id = 4003, value = "Certificate not trusted due to realm failure for principal [%s]")
-    CertificateException notTrustedRealmProblem(@Cause RealmUnavailableException e, Principal principal);
+    CertificateException notTrustedRealmProblem(@Cause Throwable e, Principal principal);
```

Comment:
@cam-rod why was this changed?

Reply:
That wasn't my change, it was from before. My guess is it was to remove the dependency on `org.wildfly.security:wildfly-elytron-auth-server` -  that module can actually be deleted from the `ssl-base` POM because it has no other uses.